src/pages/plugins/getting-started/index.md

```diff
@@ -0,0 +1,115 @@
+# Building your first plugin
+
+This tutorial will walk you through the process of building your first plugin. You will learn how to create a plugin, add a command to it, and then run it in InDesign.
+
+## Prerequisites
+
+Before you start, make sure you have the following software installed on your machine:
+
+- **InDesign 2023** or later (can be installed from the _Creative Cloud Desktop_ app)
+- **UXP Developer Tool** (can be installed from the _Creative Cloud Desktop_ app)
```

Comment:
Mention the UDT version number.
Download link from CCD: https://creativecloud.adobe.com/apps/download/uxp-developer-tools